consult-tramp.el

```diff
@@ -69,54 +79,109 @@
   :group 'consult-tramp
   :type 'string)
 
-(defcustom consult-tramp-sudo-path "/"
-  "Path to connect to the local root."
+(defcustom consult-tramp--local-hosts '("/sudo:root@localhost:/")
```

Comment:
The defcustom should probably not be private consult-tramp-local-hosts?

Reply:
No you're right, it shouldn't be